cxxopts,gulrak, and pybind11 were checked out during src_unpack to prevent
sandbox violations
diff --git a/setup.py b/setup.py
index d302525..a366a48 100644
--- a/setup.py
+++ b/setup.py
@@ -41,6 +41,10 @@ class CMakeBuild(build_ext):
         extdir = os.path.abspath(os.path.dirname(self.get_ext_fullpath(ext.name)))
         cmake_args = [
             "-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=" + str(extdir),
+            "-DFETCHCONTENT_SOURCE_DIR_CATCH2=" + str(os.environ["CATCH2_SRC_DIR"]),
+            "-DFETCHCONTENT_SOURCE_DIR_CXXOPTS=" + str(os.environ["CXXOPTS_SRC_DIR"]),
+            "-DFETCHCONTENT_SOURCE_DIR_GULRAK=" + str(os.environ["GULRAK_SRC_DIR"]),
+            "-DFETCHCONTENT_SOURCE_DIR_PYBIND11-SRC=" + str(os.environ["PYBIND11_SRC_DIR"]),
             "-DPYTHON_EXECUTABLE=" + sys.executable,
         ]
